### ISSUE TITLE
fresh clone shows unstaged change

### ISSUE THREAD

**Issue by zqfan** [hashicorp/packer#7649]
on a linux environment, with a fresh clone, and git status:
```
$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   builder/azure/arm/azure_error_response_test.TestAzureErrorNestedShouldFormat.approved.txt
	modified:   builder/azure/arm/azure_error_response_test.TestAzureErrorSimpleShouldFormat.approved.txt
	modified:   builder/azure/arm/template_factory_test.TestKeyVaultDeployment03.approved.json
	modified:   builder/azure/arm/template_factory_test.TestVirtualMachineDeployment03.approved.json
	modified:   builder/azure/arm/template_factory_test.TestVirtualMachineDeployment04.approved.json
	modified:   builder/azure/arm/template_factory_test.TestVirtualMachineDeployment05.approved.json
	modified:   builder/azure/arm/template_factory_test.TestVirtualMachineDeployment06.approved.json
	modified:   builder/azure/arm/template_factory_test.TestVirtualMachineDeployment07.approved.json
	modified:   builder/azure/arm/template_factory_test.TestVirtualMachineDeployment08.approved.json
	modified:   builder/azure/arm/template_factory_test.TestVirtualMachineDeployment09.approved.json
	modified:   builder/azure/common/template/template_builder_test.TestBuildLinux00.approved.json
	modified:   builder/azure/common/template/template_builder_test.TestBuildLinux01.approved.json
	modified:   builder/azure/common/template/template_builder_test.TestBuildLinux02.approved.json
	modified:   builder/azure/common/template/template_builder_test.TestBuildWindows00.approved.json
	modified:   examples/azure/linux_custom_image.json
	modified:   examples/azure/linux_custom_managed_image.json
	modified:   examples/azure/ubuntu_quickstart.json
	modified:   examples/azure/windows_custom_image.json
	modified:   scripts/build.ps1
	modified:   vendor/github.com/profitbricks/profitbricks-sdk-go/README.md
	modified:   vendor/github.com/sirupsen/logrus/appveyor.yml
```
I think it is caused by windows/linux new line format issue, but other projects such as terrafrom behaviors normal, here is my git settings:
```
core.editor=vim
push.default=simple
core.repositoryformatversion=0
core.filemode=true
core.bare=false
core.logallrefupdates=true
remote.origin.url=https://github.com/hashicorp/packer
remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
branch.master.remote=origin
branch.master.merge=refs/heads/master
```

**Comment by azr** [comment]
@zqfan thanks for reporting, I have seen those too, could you please open an issue with the changes ? So that we get rid of these line endings ?

**Comment by azr** [comment]
Hey there again, Megan fixed this in #7658 !
